src/main/java/ListItem.java

```diff
@@ -0,0 +1,49 @@
+public class ListItem {
+
+    private String description;
+    private String status;
+    private boolean isDone = false;
+    private String date = "";
+
+
+    public ListItem(String description, String command) {
+        try {
+            switch (command) {
+                case "todo":
```

Comment:
Clear logic flow, but perhaps you can consider separating the different task types using different classes?